### PR TITLE
Update(E2E): Make screening video playback reliable.

### PR DESCRIPTION
**Pull request by xmano**
Make video playback more reliable on E2E.

### CODE REVIEW

**Comment by xmano** [comment]
Adding some delays make the tests more reliable.

image

File: apps/festival/festival/src/app/marketplace/event/session/session.component.ts [hunk 1]
```diff
@@ -73,6 +73,7 @@ export class SessionComponent implements OnInit, OnDestroy {
         if (!!(event.meta as Screening).titleId) {
           const movie = await this.movieService.getValue(event.meta.titleId as string);
           this.screeningFileRef = movie.promotional.videos?.screener?.ref ?? '';
+          console.log("~M:>", movie, this.screeningFileRef);
```
Comment (phpgeekfr) [inline apps/festival/festival/src/app/marketplace/event/session/session.component.ts:76]: what is `"~M:>"` about ? also use "`" and not `"`

File: apps/festival/festival-e2e/src/support/pages/marketplace/FestivalMarketplaceScreeningPage.ts [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+    cy.log('>FestivalMarketplaceScreeningPage: Start Play [test-id=play]');
+    cy.get('festival-session [test-id=play]', {timeout: TO.PAGE_ELEMENT})
+      .click();
+    cy.wait(TO.ONE_SEC);
   }
 
   runVideo() {
-    cy.get('festival-session video', { timeout: 30000 }).click({ force: true });
+    cy.log('>FestivalMarketplaceScreeningPage: Play video');
+    cy.get('festival-session video', { timeout: TO.VSLOW_UPDATE })
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/support/pages/marketplace/FestivalMarketplaceScreeningPage.ts:18]: bad formatting

**Review by phpgeekfr — APPROVED** [review]

File: apps/festival/festival/src/app/marketplace/event/session/session.component.ts (hunk 1 above)
Comment (xmano) [reply apps/festival/festival/src/app/marketplace/event/session/session.component.ts:76]: Ok using single quotes and committing the change.

File: apps/festival/festival-e2e/src/support/pages/marketplace/FestivalMarketplaceScreeningPage.ts (hunk 2 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/support/pages/marketplace/FestivalMarketplaceScreeningPage.ts:18]: corrected.

**Pull request merged**